rxdogtag/gradle.properties

```diff
@@ -17,4 +17,4 @@
 POM_NAME=RxDogTag
 POM_ARTIFACT_ID=rxdogtag
 POM_PACKAGING=jar
-AUTOMATIC_MODULE_NAME=com.uber.rxdogtag
+AUTOMATIC_MODULE_NAME=com.uber.rxdogtag2
```

Comment:
Same here. Let me know if this tracks package name or coords

Reply:
I think this should just be rxdogtag2 like the package name

Reply:
Fixed 1866678